pkg/skaffold/sync/sync.go

```diff
@@ -227,9 +222,20 @@ func intersect(contextWd, containerWd string, syncRules []*latestV1.SyncRule, fi
 	return ret, nil
 }
 
-func matchSyncRules(syncRules []*latestV1.SyncRule, relPath, containerWd string) ([]string, error) {
+func matchSyncRules(syncRules []*latestV1.SyncRule, contextWd, f, containerWd string) ([]string, error) {
 	dsts := make([]string, 0, 1)
 	for _, r := range syncRules {
+		// Add directly file to destination
+		if filepath.IsAbs(f) {
+			dsts = append(dsts, path.Join(r.Dest, filepath.Base(f)))
+			continue
+		}
```

Comment:
This logic doesn't look right: if the incoming file is absolute, we should still ensure that it matches against a sync rule.  For example, a syncrule for `/etc/**/*cert*` will otherwise match `/etc/passwd`

Reply:
mark resolved